Makefile

```diff
@@ -243,9 +247,15 @@ testpkg: failpoint-enable
 ifeq ("$(pkg)", "")
 	@echo "Require pkg parameter"
 else
+ifeq ("$(case)", "")
 	@echo "Running unit test for github.com/pingcap/tidb/$(pkg)"
 	@export log_level=fatal; export TZ='Asia/Shanghai'; \
-	$(GOTEST) -ldflags '$(TEST_LDFLAGS)' -cover github.com/pingcap/tidb/$(pkg) || { $(FAILPOINT_DISABLE); exit 1; }
+	$(GOTEST) -tags 'intest' -v -ldflags '$(TEST_LDFLAGS)' -cover github.com/pingcap/tidb/$(pkg) -test.run "TestDropColumnWithUniqCompositeIndex" || { $(FAILPOINT_DISABLE); exit 1; }
```

Comment:
Please remove this debug code.

Reply:
We must add the `-tags 'intest'` other wise we cannot run any test code. And I think this should be a bug on Makefile.